src/main/java/edu/ib/Point3D.java

```diff
@@ -0,0 +1,11 @@
+package edu.ib;
+
+public class Point3D extends Point{
```

Comment:
It would be nice to override calculateDistance method from Point class since the formula differs.